src/pluginCheck.js

```diff
@@ -0,0 +1,14 @@
+// Script to check if the script uses disallowed methods and throws error if it does.
+
+function checkPlugin(namespace) {
+    let content = fetch("../plugins/" + namespace + "/plugin.js").then(res => res.text);
+    var ret = Boolean(1);
+    function testCondition(condition, errcod) {
+        if (condition) {
+            console.err(`Could not load plugin ${namespace} because of error: ${errcod}`)
+            return false;
+        }
+  }
+  [content.test(/[\0\s\n]process./), content.test(/[^"'`].*eval(/)].forEach(function(a) {a => testCondition(a, "Contains disallowed object").then(res => ret)}) // This line disallows process and eval
```

Comment:
Did you have to make a really long one-line?

Reply:
@ScolderCreations Can you maked it disallow all commands in a defined array? So that its easy to add banned commands.

Reply:
> ScolderCreations Can you maked it disallow all commands in a defined array? So that its easy to add banned commands.

Oh, yeah, sure.

Reply:
> > ScolderCreations Can you maked it disallow all commands in a defined array? So that its easy to add banned commands.
> 
> Oh, yeah, sure.

Actually, the array shown in the above snippet is of regexes to match against the plugin code, so you can use that to add commands, since every command needs a different regex. I’ll format it soon though